computation-slurm/src/main/java/com/powsybl/computation/slurm/SlurmCompletableFutureTask.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.computation.slurm;
+
+import com.powsybl.computation.CompletableFutureTask;
+
+import java.util.Objects;
+import java.util.UUID;
+import java.util.concurrent.Callable;
+
+/**
+ * @author Yichen TANG <yichen.tang at rte-france.com>
+ */
+public class SlurmCompletableFutureTask<R> extends CompletableFutureTask<R> {
```

Comment:
Note for further thought:
since the initial issue was that jsch swallows thread interruptions, we could have isolated those calls in another thread, in order to correctly catch the interruptions of the "main" task thread by cancel. It would have saved the need for this class.